src/domains/idpay-app/env/dev/terraform.tfvars

```diff
@@ -52,3 +52,9 @@ enable = {
     eventhub = true
   }
 }
+
+#
+eventhub_enrolled_pi = {
+  namespace_name      = "cstar-d-evh-ns"
+  resource_group_name = "cstar-d-msg-rg"
+}
```

Comment:
Please enable in your editor the feature that allow to put an empty row in each file